specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/operations/stable/2024-11-01/operations.json

```diff
@@ -0,0 +1,76 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2024-11-01",
+    "title": "OperationsClient",
+    "description": "Use these APIs to create extension resources through ARM, for Kubernetes Clusters."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
```

Comment:
>"paths" [](http://example.com/codeflow?start=2&length=7)

Missing path for "Operations in a Cluster" compared with base version

Reply:
The service implementation for this does not exist and returns 204 NO_CONTENT, hence this API was removed.